app/src/androidTest/java/com/example/android/architecture/blueprints/todoapp/test/chapter1/actions/RecyclerViewActionsTest.java

```diff
@@ -84,4 +92,24 @@ private void generateToDos(int count) throws Exception {
         }
         waitForElementIsGone(todoSavedSnackbar, 3000);
     }
+
+    // Chapter 2 - Exercise 6 page 53
+    @Test
+    public void refreshToDoList() throws Exception {
+        generateToDos(3);
+        onView(withId(R.id.tasks_list)).perform(swipeDown());
+    }
+
+    @Test
+    public void refreshToDoLis2() throws Exception {
+        generateToDos(3);
+        onView(withId(R.id.tasks_list)).perform(customSwipeActions.swipeCustom(5, GeneralLocation.TOP_CENTER,GeneralLocation.CENTER));
```

Comment:
Don't forget about spaces between parameters (after "," character):
`GeneralLocation.TOP_CENTER,GeneralLocation.CENTER` should be `GeneralLocation.TOP_CENTER, GeneralLocation.CENTER`